string-manipulation-1/src/get-last-char.js

```diff
@@ -0,0 +1,23 @@
+/* exported getLastChar */
+// input: string
+// output: last character of string
+
+// create storage for output
+// check if each letter is the last letter
+// if letter at current pos is last letter
+// store into output
+// return the output
+
+function getLastChar(string) {
+  var lastChar = '';
+  for (var i = 0; i < string.length; i++) {
+    if (string[i] === string[string.length - 1]) {
+      lastChar = string[i];
+    }
+  }
```

Comment:
Same thing here, no loop should be necessary since you know how to access that value without one.

Reply:
Thanks for the feedback, fixed!